src/WrapperApp/components/Results/ResultsCT.tsx

```diff
@@ -0,0 +1,34 @@
+import { Box, Button, Stack, Typography } from '@mui/material';
+import { PageCT } from '../../../JsRoot/GraphData';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import ContentCopyIcon from '@mui/icons-material/ContentCopy';
+
+export function ResultCT(props: { page: PageCT; title?: string }) {
```

Comment:
Shouldn't be this renamed to Results3D ? Then all 3D results will be displayed here